GIS/Neighborhood Names GIS.geojson

```diff
@@ -0,0 +1,5 @@
+{
```

Comment:
Can you prettify this JSON maybe?

Reply:
You can maybe move this file to `scripts/datasets` if that makes sense.

Reply:
Get rid of the change means I said you need not remove the line. You can maybe revert the change or add back the new line and add the change.